modules/preview-react/text/lib/Text.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+// type changes no need
+import {type, CanvasTypeHierarchy, CanvasTypeVariants} from '@workday/canvas-kit-react/tokens';
+import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+
+export interface TextProps extends BoxProps {
+  /**
+   * Type token as string with level and size separated by dot.
+   */
+  tokenLevel?: `${keyof CanvasTypeHierarchy}.${'large'|'medium'|'small'}`;
```

Comment:
Comment: I'm curious what the typeahead support looks like for this, but I'd have to play with it to know for sure. It might be worth us writing these out manually (as a follow-up) if the DX is significantly better.